pkg/common/si_helper.go

```diff
@@ -30,10 +30,18 @@ import (
 func createTagsForTask(pod *v1.Pod) map[string]string {
```

Comment:
Can we add a UT for this function? To cover the case where the pod has or has not owner references, or different owner references